README.md

```diff
@@ -6,7 +6,7 @@ _KRATOS Multiphysics_ ("Kratos") is a framework for building parallel, multi-dis
 
 **Kratos** is **free** under BSD-4 [license](https://github.com/KratosMultiphysics/Kratos/wiki/Licence) and can be used even in comercial softwares as it is. Many of its main applications are also free and BSD-4 licensed but each derived application can have its own propietary license.
 
-[release-image]: https://img.shields.io/badge/release-5.3-green.svg?style=flat
+[release-image]: https://img.shields.io/badge/release-6.0-green.svg?style=flat
```

Comment:
This is just one example, but please clean the PR

Reply:
Hm again after merging master into my branch and pushing, git shows all the changes from master as my own changes. This happened to me now already a few times, i don't know why...
Sorry for that, i will fix it.